test/Microsoft.ML.Predictor.Tests/TestAutoInference.cs

```diff
@@ -26,6 +26,11 @@ public TestAutoInference(ITestOutputHelper helper)
         [TestCategory("EntryPoints")]
         public void TestLearn()
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ||
+                RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                return;
+            }
             using (var env = new TlcEnvironment())
```

Comment:
maybe log an issue about re-enabling.

Reply:
Why are we disabling existing tests when we are adding new functionality?  That seems wrong.